### ISSUE TITLE
[question] Switch to candlelight / candle-usb compatible firmware

### ISSUE THREAD

**Issue by splatch** [UsbCANConverter-UCCbasic/UCCB_GS_Embedded#1]
I come across other repository which keeps gs_usb for STM32F0 family and seems to be much more active. Maybe it would be good idea to fork or contribute directly to https://github.com/candle-usb/candleLight_fw?
By this way we could have more firmware releases and hopefully even better product. :-)

**Comment by Lukasz-Juranek** [comment]
Yes you are completely right, probably next month I will try to add UCCB for  candleLight_fw.
